api/api.go

```diff
@@ -21,6 +17,36 @@ import (
 	"github.com/rubyist/tracerx"
 )
 
+// BatchOrLegacy calls the Batch API and falls back on the Legacy API
+// This is for simplicity, legacy route is not most optimal (serial)
+// TODO LEGACY API: remove when legacy API removed
+func BatchOrLegacy(objects []*ObjectResource, operation string) ([]*ObjectResource, error) {
```

Comment:
How about a note in the documentation that says announces we set "lfs.batch" to false locally if the batch operation is unsupported?